markdown/3.0/en/ff-asn.md

```diff
@@ -1,224 +1,296 @@
+```html
+<ff-asn>
+    <!-- Acts as a template for a specific group called "Category" -->
+    <ff-asn-group for-group="Category"></ff-asn-group>
 </ff-asn>
 ```
 
+**NOTE**: The `[for-group]` attribute has a higher priority than the `[filter-style]` attribute. This allows overriding certain groups even if you have a default template for that `[filter-style]`.
+
+
 ## Using native select box
+If you require a native `select` element for the **hiddenLinks** instead of the "show more/show less" implementation, you have the opportunity to define your template accordingly. Add the `select-box="true"` attribute to the `ff-asn-group` and add a `select` element with the attribute `data-container="hiddenLinks"`.
 
-In version 1.2.7 we introduced native select-box support for `ff-asn-group`.
-Additionally, the `<option>` element supports data binding when using
-`<ff-asn-group select-box="true">`. You can supply your own template
-by annotating it with the `data-template` attribute.
+You may optionally supply a template for the `option` elements. It supports data binding as usual. Make sure to mark it with the `data-template` attribute.
 
 ```html
-<ff-asn-group for-group="xxx" select-box="true">
-        ...
+<ff-asn-group for-group="YourGroupName" select-box="true">
         <div data-container="detailedLinks">
             <div data-content="detailedLinks"></div>
         </div>
 
         <select data-container="hiddenLinks">
             <option>more ...</option>
-            <!--
-                 This template is optional. If you don't supply one, it defaults to:
-                 <option>{{element.name}} {{group.unit}} ({{element.recordCount}})</option>
-            -->
+            <!-- This template is optional. If you don't supply one, it defaults to:
+                 <option>{{element.name}} {{group.unit}} ({{element.recordCount}})</option> -->
             <option data-template>My Template {{element.name}} {{group.unit}}</option>
         </select>
-        ...
 </ff-asn-group>
 ```
 
+
 ## Styling slider groups
-The only exception is the `ff-asn-group-slider`. To style slider groups
-you need to use the `ff-asn-group-slider` element.
+Slider groups a handled in a different way. To style them you need to use the `ff-asn-group-slider` element.
 
 ```html
 <ff-asn-group-slider></ff-asn-group-slider>
 ```
-**NOTE**: The `[for-group]` attribute has a higher priority than the
-`[filter-style]` attribute. This allows overriding certain groups, even
-if you have a default template for that `[filter-style]`
 
-A slider can have a `[data-container="groupCaption"]` attribute but
-instead of detailedLinks and hiddenLinks the slider group accepts a
-`ff-slider-control` and/or an `ff-slider` element.
+A slider can have a `[slot="groupCaption"]` attribute but instead of detailedLinks and hiddenLinks the slider group accepts a `ff-slider-control` and/or an `ff-slider` element.
```

Comment:
I think depending on how you pronounce _f_ you can use _a_ or _an_. For consistency we could decide now how we want to pronounce _ff_ :P

Reply:
```suggestion
A slider can have a `[slot="groupCaption"]` attribute but instead of detailedLinks and hiddenLinks the slider group accepts an `ff-slider-control` and/or an `ff-slider` element.
```